modules/deployment-model-impl/src/main/java/org/mule/runtime/module/deployment/impl/internal/policy/DefaultApplicationPolicyInstance.java

```diff
@@ -163,7 +152,7 @@ private void enableNotificationListeners(List<NotificationListener> notification
   }
 
   private void initPolicyInstance() throws InitialisationException {
-    policyInstance = policyContext.getRegistry().lookupByType(DefaultPolicyInstance.class).get();
+    policyInstance = policyContext.getRegistry().lookupByType(PolicyInstance.class).get();
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Call "Optional#isPresent()" before accessing the value. ([squid:S3655](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3655))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7629&issues=AWiGlnbUotIXnmsUonic&open=AWiGlnbUotIXnmsUonic)

[ISSUE_KEY]: AWiGlnbUotIXnmsUonic